var/spack/repos/builtin/packages/ea-utils/package.py

```diff
@@ -0,0 +1,50 @@
+##############################################################################
+# Copyright (c) 2013-2016, Lawrence Livermore National Security, LLC.
+# Produced at the Lawrence Livermore National Laboratory.
+#
+# This file is part of Spack.
+# Created by Todd Gamblin, tgamblin@llnl.gov, All rights reserved.
+# LLNL-CODE-647188
+#
+# For details, see https://github.com/llnl/spack
+# Please also see the NOTICE and LICENSE files for our notice and the LGPL.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License (as
+# published by the Free Software Foundation) version 2.1, February 1999.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+from spack import *
+
+
+class EaUtils(MakefilePackage):
+    """Command-line tools for processing biological sequencing data. Barcode
+       demultiplexing, adapter trimming, etc. Primarily written to support an
+       Illumina based pipeline - but should work with any FASTQs."""
+
+    homepage = "http://expressionanalysis.github.io/ea-utils/"
+    url      = "https://github.com/ExpressionAnalysis/ea-utils/archive/1.04.807.tar.gz"
+
+    version('1.04.807', '5972b9f712920603b7527f46c0063a09')
+
+    depends_on('subversion')
+    depends_on('zlib')
+    depends_on('gsl')
+    depends_on('bamtools')
+    # perl module required for make check
+    depends_on('perl', type='build')
```

Comment:
Correct. The build fails with this error when running `make`:
```
prove -j 4 t
Can't locate Test/Builder.pm in @INC (@INC contains: /usr/local/lib64/perl5 /usr/local/share/perl5 /usr/lib64/perl5/vendor_perl /usr/share/perl5/vendor_perl /usr/lib64/perl5 /usr/share/perl5 .) at t/join.t line 1.
BEGIN failed--compilation aborted at t/join.t line 1.
t/join.t ... 
Dubious, test returned 2 (wstat 512, 0x200)
No subtests run 
Can't locate Test/Builder.pm in @INC (@INC contains: /usr/local/lib64/perl5 /usr/local/share/perl5 /usr/lib64/perl5/vendor_perl /usr/share/perl5/vendor_perl /usr/lib64/perl5 /usr/share/perl5 .) at t/mcf.t line 1.
BEGIN failed--compilation aborted at t/mcf.t line 1.
t/mcf.t .... 
Dubious, test returned 2 (wstat 512, 0x200)
No subtests run 
Can't locate Test/Builder.pm in @INC (@INC contains: /usr/local/lib64/perl5 /usr/local/share/perl5 /usr/lib64/perl5/vendor_perl /usr/share/perl5/vendor_perl /usr/lib64/perl5 /usr/share/perl5 .) at t/multx.t line 1.
BEGIN failed--compilation aborted at t/multx.t line 1.
t/multx.t .. 
Dubious, test returned 2 (wstat 512, 0x200)
No subtests run 

Test Summary Report
-------------------
t/join.t (Wstat: 512 Tests: 0 Failed: 0)
  Non-zero exit status: 2
  Parse errors: No plan found in TAP output
t/mcf.t  (Wstat: 512 Tests: 0 Failed: 0)
  Non-zero exit status: 2
  Parse errors: No plan found in TAP output
t/multx.t (Wstat: 512 Tests: 0 Failed: 0)
  Non-zero exit status: 2
  Parse errors: No plan found in TAP output
Files=3, Tests=0,  1 wallclock secs ( 0.02 usr +  0.00 sys =  0.02 CPU)
Result: FAIL
make: *** [check] Error 1
```

In the `Makefile`, we have:
`all: $(BIN) check` at the beginning, and later in the file:
`check: $(BIN)
        prove -j 4 t` of which will not work without perl